src/include/btmem.h

```diff
@@ -77,6 +77,7 @@ struct __wt_page_header {
 #define WT_PAGE_EMPTY_V_NONE 0x04u /* Page has no zero-length values */
 #define WT_PAGE_ENCRYPTED 0x08u    /* Page is encrypted on disk */
 #define WT_PAGE_UNUSED 0x10u       /* Historic lookaside store page updates, no longer used */
+#define WT_PAGE_FT_UPDATE 0x20u    /* Page contains updated fast-truncate information */
```

Comment:
Alphabetize

Reply:
I didn't alphabetize this because these bit values are persistent and can never change, that is, they are sorted on bit order rather than name. If I alphabetize them, then the bit values will be out of order.

Reply:
They can't be, because they are written to disk we need to keep the existing values the same.

Reply:
No need to change the flag value. Just reorder the entries. WT_PAGE_UNUSED actually has the smallest value.